Week3/homework/index.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const express = require('express');
+const mysql = require('mysql');
+const bodyParser = require('body-parser');
+
+const app = new express();
+
+app.use(bodyParser.json());
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'todo_database',
+});
+
+connection.connect();
+
+app.put(`/createList/:list`, (req, res) => {
+  const { list } = req.params;
+  connection.query(
+    `CREATE TABLE ${list} (todo_id INT(11) NOT NULL AUTO_INCREMENT,  todo varchar(50), done enum("true", "false"), user_id INT,  PRIMARY KEY (todo_id)  );`,
```

Comment:
What is the purpose of asking the user the name of the table? you should have a table list containing a user_id which is a foreign key on user table, this way you know to which user belongs the list.
You could have a list table like you have with todo and fill it with insert statement.